### DISCUSSION TITLE
Is it true?

### DISCUSSION THREAD

**Discussion by Aspie96** [rms-support-letter/revoke-open-letter-signature#4] [General]
Is it actually true that people are unable to revoke their signature? And do we have evidence

**Comment by nukeop** [comment]
Yes, there might be dozens of such people. We have been in contact with some of those people.

**Comment by nukeop** [comment]
https://github.com/rms-support-letter/rms-support-letter.github.io/pull/6658

Evidence that it's impossible to revoke a signature on rms-open-letter

**Comment by Aspie96** [comment]
I think those removing the signature should open a PR in the open letter repo and then archive it trough archive.is or web.archive.org so that there is evidence of this.

Could a request to do so be added in the readme?

**Comment by Aspie96** [comment]
Ah I see, they cannot create a PR!

But maybe they could ask via e-mail?

**Comment by nukeop** [comment]
The email is no longer monitored. So there's quite literally no way to do it.

**Comment by Aspie96** [comment]
Still this can be seen badly:
> Some people have asked to remove their signatures from the letter against Stallman and the FSF, and the people who run it won’t let them.

More neutral tone:
> Some people have asked to remove their signatures from the letter against Stallman and the FSF, but the letter is now closed and won't accept contributions, making removing ones signature impossible.

**Comment by nukeop** [comment]
https://lbry.tv/@Lunduke:e/StallmanLetterLocked:f

More evidence

**Comment by Aspie96** [comment]
Now that @rany2 got his signature removed does this repo make sense?

I see no evidence of people being unable to remove the signature. Not many people were actually removing their signature

(The video simply states it)

**Reply by rany2** [reply]
It's more like removing your signature is a bit of hassle because it involves sending them an email.

**Reply by Aspie96** [reply]
@rany2 but it looks like one can open PRs now.

Was it impossible then?

If it was impossible and now it is possible this letter no longer makes sense IMHO.

In fact if the claim is now false it must be removed

**Reply by rany2** [reply]
![image](https://user-images.githubusercontent.com/71077389/114735827-38bf4080-9d4e-11eb-85fa-e5f74b28a206.png)

**Reply by rany2** [reply]
test is from now but 

> Because your signature wasn't merged as a PR (you were in the very first batch of signers when I was doing it in batches), you weren't able to remove your signature as we limited access to the repo to reduce spam. I'm removing you here, please feel free to reach out directly if you need anything else.

So according to her I can't make a PR because i didn't get a PR merged to the repo. Nothing changed the only difference is that those people made a PR and had it merged

**Reply by rany2** [reply]
Nothing changed but it seems like anyone who made a PR and had it merged would have been able to remove their signature all this time

**Reply by Aspie96** [reply]
@rany2 did you originally sign trough PR or by email? I understand no PR from you was merged but did you originally sign with one?
If so, do you have a link to your signing PR?

(Thank you as always for the info you provide BTW).

**Comment by Aspie96** [comment]
After reviewing evidence it seems that:

- Not many people at all tried to remove their signature. Some of the few removals are the admins removing trolls or duplicates.
- It is possible to open PRs but only for contributors. So essentially only those who signed trough PR can remove their signature and only if it was accepted trough PR and not added manually.
- Not all signatures provided trough PR were merged trough PR. So there are people who signed trough PR but cannot remove their signature trough PR.
- Some seem to believe the number of signatures was clearly and strongly declining. If that was the case many people would be opening PRs so it doesn't seem to be the case.
- We do not know why @rany2's signature was removed. So far we have no idea whether they actually monitor the email address or not. It is very possible they found out about his removal trough this repo and public discussion with rany.

Overall this statement:
> the people who run it won’t let them

Might be true. But it might also not be. It depends on whether they monitor the email address or not.
So far we have not one example of anybody who is unable to remove their signature.

I think the statement needs to be removed since it is potentially misleading

**Reply by rany2** [reply]
There was no public discussion with them and they didn't reply directly to my email

**Reply by rany2** [reply]
I think there is a chance they didn't see my email but instead checked the repo

**Reply by Aspie96** [reply]
Did you originally sign via PR (even though they didn't merge it)?

**Reply by rany2** [reply]
Yes, they closed my PR

**Reply by rany2** [reply]
https://github.com/rms-open-letter/rms-open-letter.github.io/pull/256

**Reply by Aspie96** [reply]
I edited my post to reflect the new evidence, thank you!

**Reply by Aspie96** [reply]
Still this statement:
> the people who run it won’t let them

Is potentially false or at least there is not enough evidence for it.
